Jenkinsfile

```diff
@@ -30,7 +30,7 @@ def targets = [
   "NUCLEO_F401RE": ["ATMEL", "MCR20A"],
   "NUCLEO_F429ZI": ["ATMEL", "MCR20A", "S2LP"],
   //"NCS36510": ["internal"],
-  "UBLOX_EVK_ODIN_W2": ["ATMEL"],
+  //"UBLOX_EVK_ODIN_W2": ["ATMEL"],
```

Comment:
should we just remove whole line

Reply:
Previous MCU was already commented, following the same pattern.